index.d.ts

```diff
@@ -128,8 +128,9 @@ export interface DailyCallOptions {
   cssText?: string;
   dailyConfig?: DailyAdvancedConfig;
   subscribeToTracksAutomatically?: boolean;
-  videoSource?: string | MediaStreamTrack;
-  audioSource?: string | MediaStreamTrack;
+  videoSource?: boolean | MediaStreamTrack;
+  audioSource?: boolean | MediaStreamTrack;
```

Comment:
I believe it's still necessary to support a string in both `videoSource` and `audioSource` here (in addition to the `boolean` and `MediaStreamTrack`), as the documentation states:
> Pass a device ID string (like you get back from enumerateDevices()) to specify what audio input device to use.

Reply:
Good point.  In that case the [API docs](https://docs.daily.co/reference#properties) should be updated to reflect that as well:
![image](https://user-images.githubusercontent.com/3887189/119570184-a2c30080-bd64-11eb-9a57-ff00f1335f49.png)
![image](https://user-images.githubusercontent.com/3887189/119570212-a9517800-bd64-11eb-9ab8-ecc7cb3c5220.png)